sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -9,12 +9,72 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     this.listenTo(this.categories, 'sync change:unread_count', this.render);
   },
 
+  events: {
+    'click #login-submit': 'loginSubmit',
+  },
+
+  loginSubmit: function(e){
+    e.preventDefault()
+
+    var formData = JSON.stringify($("#login").serializeObject());
+
+    $.ajax({
+      type: "POST",
+      url: "/login",
+      data: formData,
+      success: function(response){
+        var user = response.response.user;
+        if (user) {
+          location.reload();
+        }
+        var errors = response.response.errors;
+        if (errors) {
+          errorMsg = errors.email || errors.password || errors.rememeber || {};
+          $("#login-errors").html(errorMsg);
+          $('#password').val('');
+        }
+        return;
+      },
+      contentType : "application/json"
+    });
+  },
+
   render: function() {
     activeTab = $(".tab .active").text().toLowerCase() || "categories"
 
     var content = this.template({active: activeTab, current_user: this.user, feeds: this.feeds, categories: this.categories})
-    $('#nav-mobile').html(content);
+    this.$el.html(content);
 
     $('ul.tabs').tabs();
+    return this;
   },
 });
+
+// Source: http://stackoverflow.com/questions/1184624/convert-form-data-to-js-object-with-jquery
```

Comment:
We're overriding jQuery, so if a page includes "sidenav_view.js," jQuery is changed for all other js files. Since we're overriding the global jQuery object, it's weird to only keep it in this one file. That's why I think we should either move the function or remove jQuery.fn from the name.